### ISSUE TITLE
Fatal Exception: java.lang.IllegalArgumentException: bad base-64

### ISSUE THREAD

**Issue by puboe** [Adyen/adyen-3ds2-android#52]
**Describe the bug**
After updating to version `2.2.11` of the SDK we started getting `java.lang.IllegalArgumentException - bad base-64` crashes in Crashlytics. The exception seems to be happening inside the SDK's codebase after the call to `Transaction.doChallenge(...)`. Apart from bumping the SDK's version nothing else has changed in out 3DS flow.

**Smartphone (please complete the following information):**
 - Device: Any
 - OS: Android
 - Version: Any 

**Additional context**
This is a sample stacktrace. I reckon it might not be very useful as it's obfuscated

```
# Crashlytics - Stack trace
# Application: com.glovo
# Platform: android
# Version: 5.208.2 (268230)
# Issue: 658474b622c11c5f6042d479d0b1e384
# Session: 6426DBB00343000142382BDC2E3EC369_DNE_3_v2
# Date: Fri Mar 31 2023 15:16:11 GMT+0200 

Fatal Exception: java.lang.IllegalArgumentException: bad base-64
       at android.util.Base64.decode(Base64.java:163)
       at android.util.Base64.decode(Base64.java:138)
       at atd.s0.a.b(a.java:2)
       at atd.s0.a.a(a.java:9)
       at atd.h0.a.<init>(a.java:11)
       at atd.h0.b.a(b.java:6)
       at atd.g.a.<init>(a.java:2)
       at com.adyen.threeds2.internal.i.a(i.java:12)
       at com.adyen.threeds2.internal.i.doChallenge(i.java:11)   <--- transaction.doChallenge(...)
       at app.package.redacted.doChallenge(SomeClass.kt:124)
       at com.processout.processout_sdk.CustomerActionHandler.handleCustomerAction(CustomerActionHandler.java:61)
       at com.processout.processout_sdk.ProcessOut$4.onSuccess(ProcessOut.java:335)
       at com.processout.processout_sdk.ProcessOut$14.onSuccess(ProcessOut.java:894)
       at com.processout.processout_sdk.Network$1.onResponse(Network.java:63)
       at com.processout.processout_sdk.Network$1.onResponse(Network.java:60)
       at com.android.volley.toolbox.JsonRequest.deliverResponse(JsonRequest.java:100)
       at com.android.volley.ExecutorDelivery$ResponseDeliveryRunnable.run(ExecutorDelivery.java:102)
       at android.os.Handler.handleCallback(Handler.java:942)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loopOnce(Looper.java:240)
       at android.os.Looper.loop(Looper.java:351)
       at android.app.ActivityThread.main(ActivityThread.java:8355)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:584)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1013)
```

**Comment by tkuntubayev** [comment]
Hi @puboe,
Thank you for raising this issue!

Our team will investigate it. We will come back soon if extra details will be needed.

**Comment by fdcb** [comment]
Hey @puboe this issue has been fixed in the newest version 2.2.14.
Thank you for flagging this!
